packages/cdc-react/src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,85 @@
+@use "../../scss/global.scss";
+@use "uswds-core" as *;
+
+.cdc-react {
+  .usa-alert.fluid {
+    max-width: initial;
+  }
+
+  .usa-alert.left-align .usa-alert__body {
+    margin-left: 0;
+  }
+
+  .usa-alert {
+    border-left: none;
+    border-top-width: 0.5rem;
+    border-top-style: solid;
+    border-radius: 4px 4px 0 0;
+    line-height: 140%;
+    max-width: 64rem;
```

Comment:
Can we see about utilizing the units mixin here?

Reply:
yea, good call, ill update